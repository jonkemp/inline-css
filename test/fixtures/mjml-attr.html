<style>
    .mj-section {
        full-width: 'full-width';
        text-align: center;
        background-color: tomato;
    }
    .mj-text {
        font-family: 'Inconsolata', monospace;
        color: #333;
    }
</style>
<mjml>
  <mj-body>
    <mj-section>
      <mj-text>This is a label.</mj-text>
    </mj-section>
    <mj-section class="mj-section">
      <mj-text class="mj-text">This is a label.</mj-text>
    </mj-section>
  </mj-body>
</mjml>
