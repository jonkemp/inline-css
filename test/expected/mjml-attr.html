
<mjml>
  <mj-body>
    <mj-section>
      <mj-text>This is a label.</mj-text>
    </mj-section>
    <mj-section class="mj-section" style="" full-width="full-width" text-align="center" background-color="tomato">
      <mj-text class="mj-text" style="" font-family="'Inconsolata', monospace" color="#333">This is a label.</mj-text>
    </mj-section>
  </mj-body>
</mjml>
